noodles-editor/public/noodles/california-earthquakes/README.md

```diff
@@ -0,0 +1,40 @@
+# California Earthquakes
+
+## Overview
+This example visualizes earthquake data for California, where the size of each point represents earthquake magnitude and colors represent intensity using a color ramp.
+
+## What It Demonstrates
+- **Data-driven sizing**: Circle radius scales with earthquake magnitude using square root for better visual perception
+- **Color ramps**: Using `ColorRampOp` to map magnitude values to a color gradient
+- **Value mapping**: `MapRangeOp` normalizes magnitude values for color mapping
+- **Accessor expressions**: JavaScript expressions to transform data values
+
+## Key Techniques
+- **Data source**: `FileOp` loads earthquake data from CSV
+- **Position accessor**: Extracts `[Longitude, Latitude]` from data
+- **Magnitude accessor**: Extracts magnitude value for both radius and color calculations
+- **Radius calculation**: `Math.sqrt(d.Magnitude) * 15` creates perceptually better sizing
+- **Value normalization**: `MapRangeOp` scales magnitude from 0-5 range to 0-1 for color ramp
```

Comment:
Do you also feel this section and "what is demoed" are repeating points, and can be combined? Trying to figure out what motivates the split.

If we want to two sections, maybe one is focused on saying it in plain english while the other maps it to implementation? But it can be combined and remain concise I think that's easier to comprehend